theme/src/components/do-dont.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {Flex, CircleOcticon, Text, Box} from '@primer/components'
+import Caption from './caption'
+import {X, Check} from '@primer/octicons-react'
+
+export const DoDontWrapper = ({children}) => {
+  return (
+    <Flex flexDirection={['column', 'row', 'row']}>
+      {children}
+    </Flex>
+  )
+}
+
+export const Do = ({src, children}) => {
+  return (
+    <Flex flexDirection='column' mr="2">
+      <Flex alignSelf='start' flexDirection="row" alignItems="center" mb="2">
+        <CircleOcticon icon={Check} size={16} bg="green.5" color="white" mr="2" p="1"/>
+        <Text fontWeight="bold" color="gray.9" ml="1">
+          Do
+        </Text>
+      </Flex>
+      <Box as='img' src={src} width="100%" height="auto"/>
```

Comment:
I generally don't love passing in text content as props as it feels pretty awkward 🤔 The other alternative would be to allow users to pass in both the `img` and `Caption` as children, but we have to apply the `width` and `height` properties to the `img` tag which means we'd need to do some cloning and editing of children which I generally try to avoid. I think it's fine to leave this minimal for now and if it starts causing issues we can rethink the API in the future